includes/data-stores/class-wc-product-variable-data-store-cpt.php

```diff
@@ -75,7 +75,8 @@ protected function read_attributes( &$product ) {
 					$options = wc_get_text_attributes( $meta_value['value'] );
 				}
 
-				$attribute = new WC_Product_Attribute();
+				$classname = apply_filters( 'woocommerce_product_attribute_class', 'WC_Product_Attribute', $product );
```

Comment:
Should probably call this filter `classic_commerce_product_attribute_class` or `cc_product_attribute_class` instead, unless this change is also intended to be incorporated into WooCommerce. (If so, then it should be submitted to Woo first.)